README.md

```diff
@@ -90,7 +90,7 @@ Snapshot version:
 	<dependency>
 		<groupId>fr.inria.gforge.spoon</groupId>
 		<artifactId>spoon-core</artifactId>
-		<version>6.0.0-SNAPSHOT</version>
+		<version>7.0.0-SNAPSHOT</version>
```

Comment:
should be 6.1.0

Reply:
I hesitated but as CtImport is almost ready, it will be immediately a new major release of Spoon... So IMHO we won't have another minor release before Spoon 7.0.0.